### PR TITLE
Migration/load query pane to react

### PR DESCRIPTION
**Pull request by sunilyadav840**
Created LoadQueryPanel/index.tsx (React component for LoadQueryPanel panel).
Cleanup knockout LoadQueryPanel code(removed LoadQueryPane.ts and LoadQueryPanel.html)

### CODE REVIEW

File: src/Explorer/Panes/LoadQueryPanel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        }
+      )
+      .finally(() => {
+        setLoadingFalse();
+        clearInProgressMessageWithId(id);
+      });
+  };
+
+  const loadQueryFromFile = async (file: File): Promise<void> => {
+    const selectedCollection: Collection = explorer && explorer.findSelectedCollection();
```
Comment (languy) [inline src/Explorer/Panes/LoadQueryPanel/index.tsx:94]: nit: use optional chaining where possible:
```suggestion
    const selectedCollection: Collection = explorer?.findSelectedCollection();
```

**Review by languy — APPROVED** [review]

File: src/Explorer/Panes/LoadQueryPanel/index.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+import QueryTab from "../../Tabs/QueryTab";
+import { Collection } from "..//../../Contracts/ViewModels";
+import { GenericRightPaneComponent, GenericRightPaneProps } from "../GenericRightPaneComponent";
+
+interface LoadQueryPanelProps {
+  explorer: Explorer;
+  closePanel: () => void;
+}
+
+export const LoadQueryPanel: FunctionComponent<LoadQueryPanelProps> = ({
```
Comment (languy) [inline src/Explorer/Panes/LoadQueryPanel/index.tsx:25]: Can we get some unit test on this component? Maybe an enzyme snapshot?

File: src/Explorer/Panes/LoadQueryPanel/index.tsx (hunk 1 above)
Comment (sunilyadav840) [reply src/Explorer/Panes/LoadQueryPanel/index.tsx]: updated

File: src/Explorer/Panes/LoadQueryPanel/index.tsx (hunk 2 above)
Comment (sunilyadav840) [reply src/Explorer/Panes/LoadQueryPanel/index.tsx:25]: added snapshot.

File: src/Explorer/Panes/LoadQueryPanel/index.tsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+      setFormErrorsDetails("No file specified. Please input a file.");
+      logConsoleError("Could not load query -- No file specified. Please input a file.");
+      return;
+    }
+
+    const file: File = selectedFiles[0];
+    const id: string = logConsoleMessage(ConsoleDataType.InProgress, `Loading query from file ${file.name}`);
+    logConsoleProgress(`Loading query from file ${file.name}`);
+    setLoadingTrue();
+    loadQueryFromFile(file)
```
Comment (southpolesteve) [inline src/Explorer/Panes/LoadQueryPanel/index.tsx:75]: Use async/await here

Comment (sunilyadav840) [reply src/Explorer/Panes/LoadQueryPanel/index.tsx]: Updated async await

**Review by southpolesteve — APPROVED** [review]

**Pull request merged**